src/frontend/components/header/header.html

```diff
@@ -1,42 +1,76 @@
 <div class="flex flex-column flex-stretch col-12">
-  <header class="bg-darken-1 border-bottom py3 px3">
-    <h4 class="regular flex flex-center primary-color col col-4 m0">
+  <header class="flex flex-justify bg-panel border-bottom py3 px3 dark-theme">
```

Comment:
I can't see where this .dark-theme style is defined - is it defined? It looks like the right thing to use is "dark" or "light", not "dark-theme" - or am I misunderstanding?